### DISCUSSION TITLE
How AlertManager fit in this project?

### DISCUSSION THREAD

**Discussion by Dentrax** [aquasecurity/starboard#887] [Development]
The idea is built-in support for AlertManager in this project.

```
$ kubectl get vulnerabilityreports.aquasecurity.github.io

NAME                                                REPOSITORY                                     TAG      SCANNER   AGE
pod-dnsutils-dnsutils                               e2e-test-images/jessie-dnsutils                1.3      Trivy     18m
```

When a new `VulnerabilityReport` occured/changed/deleted, we can fire an alert to AlertManager:

```
Cluster foo comprimised for pod-dnsutils-dnsutils
Alert: T
Description: {x} vulnerabilities found
Details:
 - Scanner: Trivy (v1)
 - VulnDB: v2
 - Tag: 1.3
 - Repository: e2e-test-images/jessie-dnsutils
 ...
```

For example implementation: https://github.com/prometheus-operator/prometheus-operator/tree/main/pkg/alertmanager

We can add an `alertmanager` config in the helm chart: https://github.com/aquasecurity/starboard/blob/main/deploy/helm/values.yaml

Any thoughts? Does it make sense? 🤔

**Comment by danielpacak** [comment]
IMO It fits better in scope of https://github.com/giantswarm/starboard-exporter or a similar tool that's a consumer of data generated by Starboard. But I'm curious what other community members think!
